components/dashboard/src/settings/SelectIDE.tsx

```diff
@@ -176,42 +176,40 @@ function renderIdeOption(
     version: IDEOption["imageVersion"],
     onSelect: () => void,
 ): JSX.Element {
-    const label = option.type === "desktop" ? "" : option.type;
+    const shouldShowOptionType = option.type !== "desktop" || option.title === "VS Code"; // Force show of "Desktop" in the list for VS Code Desktop
     const card = (
         <SelectableCardSolid className="w-36 h-44" title={option.title} selected={selected} onClick={onSelect}>
-            <div className="flex justify-center mt-3">
-                <img className="w-16 filter-grayscale self-center" src={option.logo} alt="logo" />
-            </div>
-            <div
-                className="mt-2 px-3 py-1 self-center"
-                style={{
-                    minHeight: "1.75rem",
-                }}
-            >
-                {label ? (
-                    <span
-                        className={`font-semibold text-sm ${
-                            selected ? "text-gray-100 dark:text-gray-600" : "text-gray-600 dark:text-gray-500"
-                        } uppercase`}
-                    >
-                        {label}
-                    </span>
-                ) : (
-                    <></>
-                )}
-            </div>
-
             {version ? (
-                <div
-                    className={`font-semibold text-xs ${
-                        selected ? "text-gray-100 dark:text-gray-600" : "text-gray-600 dark:text-gray-500"
-                    } uppercase px-3 self-center`}
+                <span
+                    className={`font-normal text-xs ${
+                        selected
+                            ? "text-gray-100 dark:text-gray-600"
+                            : "text-gray-500 contrast-more:text-gray-600 dark:text-gray-500"
+                    } pl-1 self-start`}
                     title="The IDE's current version on Gitpod"
                 >
```

Comment:
**suggestion**: Could we tone down the font color one scale point?

Reply:
Will do, also will leave the version with more contrast as is, because for some folks this can really be impossible to read so we can let them if they prefer.